src/jetson/main.py

```diff
@@ -390,59 +87,59 @@ def drawFrame(boxes, frame, fps):
 
 if __name__ == "__main__":
     warnings.filterwarnings("once")
-    parser = argparse.ArgumentParser(description="Face detection")
-    parser.add_argument('--detector', '-d', type=str, required=True,
-                        help="Path to a trained face detector .pth file")
-    parser.add_argument('--detector_type', '-t', type=str, required=True, help="Type of face detector. One of "
-                                                                               "blazeface, ssd, or retinaface.")
-    parser.add_argument('--cuda', '-c', default=False,
-                        action='store_true', help="Enable cuda")
-    parser.add_argument('--classifier', type=str,
-                        help="Path to a trained classifier .pth file")
-    parser.add_argument('--write_imgs', default=False,
-                        help='Write images to output_dir')
-    parser.add_argument('--output_dir', default='encrypted_imgs',
-                        type=str, help="Where to output encrypted images")
-    args = parser.parse_args()
 
-    if args.detector_type not in DETECTOR_TYPES:
-        print(
-            'Please include a valid detector type (\'blazeface\', \'ssd\', or \'retinaface\'')
+    with open('src/jetson/config.json') as file:
```

Comment:
This will only work if `main.py` is run from the `Embedded2` folder. Use             `open(os.path.join(os.path.dirname(__file__), 'config.json')) as file` instead

Reply:
Thanks. I've implemented this in both `main.py`, and `db/config.py` similarly for `login.json`